src/main/java/reposense/RepoSense.java

```diff
@@ -28,6 +28,8 @@
 import reposense.util.FileUtil;
 
 public class RepoSense {
+    public static final String DEFAULT_VERSION = "v1.8rc";
```

Comment:
Use a generic value so we don't have to maintain this.

The only time the value will be used is when the Jar is built without `-Pversion`.

Reply:
Can you suggest one?

Reply:
Or just empty string?

Reply:
Empty string is good.

Having a version may be confusing on Netlify previews.